Code/Tools/FBuild/FBuildCore/BFF/Functions/Function.cpp

```diff
@@ -524,6 +525,51 @@ bool Function::GetDirectoryListNodeList( NodeGraph & nodeGraph,
     return true;
 }
 
+// GetCompilerNode
+//------------------------------------------------------------------------------
+bool Function::GetCompilerNode(NodeGraph & nodeGraph, const BFFIterator & iter, const AString & compiler, CompilerNode * & compilerNode) const
```

Comment:
Moved GetCompilerNode to Function base class as it is now used in CSNode, ObjectNode, and ObjectListNode.